Wireframe/index.html

```diff
@@ -4,10 +4,43 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <header>
+      <h1>Wireframe Project</h1>
+      <p>Plan first then Develop!</p>
+    </header>
+
+    <div class="main-article">
+      <img src="./img/GitHub-headpic.webp" alt="Main Article Image">
```

Comment:
1. Git  allows multiple developers to track changes in code, collaborate on projects, and manage the history of a project’s code
2. Developers need Git for Version Control, Collaboration, Integration and Backup.
3. A branch in Git is essentially a parallel version of the project. It allows you to work on new features, bug fixes, or other changes independently from the main code.

I will work on creating new pages to explain more in details about these questions.

Regarding the change with <img> I want to be honest, I have no idea what you meant because my code already has the "alt" for all images.